cores/epoxy/main.cpp

```diff
@@ -15,9 +15,9 @@
  * inserts it into the Serial buffer.
  */
 
-#if defined(EPOXY_DUINO) && __unix__
-
 #include "Arduino.h"
+
+#if __has_include(<termios.h>)
```

Comment:
This `__has_include()` is a new one for me. As far as I can tell, it was only recently added to the C++ standard (C++17?). MSVC added support for it only in 2017 (https://docs.microsoft.com/en-us/cpp/preprocessor/hash-if-hash-elif-hash-else-and-hash-endif-directives-c-cpp?view=msvc-170#__has_include). Maybe gcc/g++ had support for it for a while, but it was probably non-standard. Arduino uses C++11, so if we pass a `-std=c++11` flag to the compiler, will this work? I'm not sure..

Let's just stick with the simple `#if defined(__MINGW32__)` because I *know* that this is supported everywhere.